index.html

```diff
@@ -0,0 +1,220 @@
+          <p class="program-description">
+            Thematic development in story. Planning and execution in story. How
+            to identify the themes in self-created writing
+          </p>
+        </div>
+        <div class="program-card">
+          <img
+            id="workshop-icon"
+            class="workshop-icon"
+            src="./images/workshop-icon.svg"
+            alt="workshop-icon"
+          />
+          <h3 class="program-title">Workshop</h3>
+          <p class="program-description">
+            Apply what you learned. Start creating your story and plan it for
+            brief presentation to peers and professors.
+          </p>
+        </div>
+        <div class="program-card">
+          <img
+            id="network-icon"
+            class="network-icon"
+            src="./images/network-icon.svg"
+            alt="network-icon"
+          />
+          <h3 class="program-title">Present</h3>
+          <p class="program-description">
+            Present your work and get professional feedback from top tier
+            professors. Network with your peers and professors.
+          </p>
+        </div>
+        <div></div>
+      </div>
+      <div class="button-wrapper">
+        <button class="join-button">Join Cohort Course 2022</button>
+      </div>
+
+      <a href="#" class="program-link">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section class="featured-speakers">
+      <h2 class="speakers">Featured Speakers</h2>
+      <div class="red-line-container">
+        <img class="red-line" src="./images/red-line.png" alt="red-line" />
+      </div>
+      <div class="speakers-container"></div>
+    </section>
+
+    <section class="partners">
+      <div class="partner-title">Partner</div>
+      <div class="red-line-container">
+        <img class="red-line" src="./images/red-line.png" alt="red-line" />
+      </div>
+      <div class="partners-container">
+        <img
+          class="mozilla-logo"
+          src="./images/mozilla-logo.png"
+          alt="mozilla-logo"
+        />
+        <img
+          class="google-logo"
+          src="./images/google-logo.png"
+          alt="google-logo"
+        />
+        <img
+          class="naver-logo"
+          src="./images/naver-logo.png"
+          alt="naver-logo"
+        />
+        <img
+          class="daumkakao-logo"
+          src="./images/daumkakao-logo.png"
+          alt="daumkakao-logo"
+        />
+        <img
+          class="airbnb-logo"
+          src="./images/airbnb-logo.png"
+          alt="airbnb-logo"
+        />
+      </div>
+    </section>
+    <footer class="footer">
+      <div class="footer-logo-wrapper">
+        <img class="footer-logo" src="./images/logo2.png" alt="footer-logo" />
+      </div>
+      <div class="footer-description">
+        <p class="footer-title">2022 Creative Writing Course</p>
+        <p class="rights">Some Rights Reserved</p>
+      </div>
+    </footer>
```

Comment:
- [ ] Kindly increase the padding-top and padding-bottom  of your footer in order to produce a close replica of the guiding template.

### Your Design

![your-footer](https://user-images.githubusercontent.com/26597432/196836592-0e416bdd-a71a-435e-99b6-2f656ea32488.png)


### Guiding Template

![temp-footer](https://user-images.githubusercontent.com/26597432/196836633-4d29b670-b49a-453b-8253-080721b23cfa.png)

Reply:
- [x] This issue is well resolved :heavy_check_mark: